src/dfuTrigger.js

```diff
@@ -191,6 +194,8 @@ const sendDetachRequest = openDecorator((usbdev, interfaceNumber) => (
                         // This edge case only happens when using the "libusb" kernel
                         // driver on win32 (not "winusb", not "libusbk")
                         resolve();
+                    } else if (os.platform() === 'darwin') {
```

Comment:
I'm wondering if on mac resolve should happen only when no error happened. Would it be better to say:
`if (!err && os.platform() === 'darwin') ...` ?

Reply:
Sure, this could catch the error on MacOS.